functions/controllers/post/newTopic.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable max-len */
+// const firebase = require('firebase');
+const HttpStatus = require('http-status-codes');
+// const { validateCode } = require('../../util/validator');
+const { status, message } = require('../../util/constants');
+
+const { error, success } = status;
+const {
+    somethingWentWrong,
+    topicExists,
+} = message;
+const {
+    INTERNAL_SERVER_ERROR,
+    CREATED,
+    CONFLICT,
+} = HttpStatus;
```

Comment:
const { BAD_REQUEST, OK, INTERNAL_SERVER_ERROR } = require('http-status-codes');
preferred approach now

Reply:
also fixing the httpstatus code import would resolve the conflict